addons/vectors/fnc_vectorRotate3D.sqf

```diff
@@ -0,0 +1,34 @@
+/*
```

Comment:
Header style doesn't match rest of CBA
https://github.com/CBATeam/CBA_A3/blob/master/addons/vectors/fnc_vectRotate2D.sqf

Reply:
A better header with examples and such:
https://github.com/CBATeam/CBA_A3/blob/master/addons/settings/fnc_init.sqf